.github/workflows/ziggurat-release-event.yml

```diff
@@ -0,0 +1,35 @@
+name: ziggurat-release-event
+
+on:
+  repository_dispatch:
+    types: [ziggurat-release-event]
+
+jobs:
+  changelog-copy:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout Ziggurat repo
+        uses: actions/checkout@v2
+        with:
+          repository: shubhang93/ziggurat #TODO change it to gojek/ziggurat
```

Comment:
lets change to gojek/ziggurat, what's stopping us?

Reply:
this was just for testing it,  once we obtain the credentials then we can change it, or else the pipelines will keep failing